eq-author-api/repositories/QuestionConfirmationRepository.js

```diff
@@ -45,12 +49,28 @@ module.exports = knex => {
       .restore(id)
       .then(head);
 
+  const getPipingAnswers = async id => {
+    const { pageId } = await findById(id);
+    return PreviousAnswerStrategy(knex).getPreviousAnswersForPage({
+      id: pageId,
+      includeSelf: true,
```

Comment:
Perhaps I misunderstood how this flag works, but when I tested e2e i wasn't able to select answers on the page that the confirmation page hangs off. I set something up like this (note I created Q2 after Q1):

- S1
  - Q2
    - Answer 2
  - Q1
    - Answer 1
    - Confirmation page

On the confirmation page, I was able to pipe answer 1 but not answer 2. Is that expected?

Reply:
No thats not intentional. The include self should include it in the list of pages. I'll look into it.

Reply:
Is your branch up to date this was resolved and I cannot recreate it locally. I also have a cypress test that covers it.

Reply:
Apologies, looks like my branch didn't have the latest changes. After a pull this worked as expected for me. 👍